### ISSUE TITLE
[LOW] Wrong outdated message for HA-Proxy 1.8+

### ISSUE THREAD

**Issue by zib92130** [LetheanMovement/lethean-vpn#11]
During requirements checking , I get this :
```
Found HAPROXY_BIN at /usr/sbin/haproxy
Your haproxy is outdated! You need at least 1.6 version:
HA-Proxy version 1.8.8-1ubuntu0.1 2018/05/29
Copyright 2000-2018 Willy Tarreau <willy@haproxy.org>
```
 
Quick fix, probably not optimum for upcoming version, in configure.sh to add version 1.8 :
```
51c51
<     if ! $HAPROXY_BIN -v | grep -qE "version 1.8|version 1.7|version 1.6"; then
---
>     if ! $HAPROXY_BIN -v | grep -qE "version 1.7|version 1.6"; then
```

**Comment by valiant1x** [comment]
Appreciate the info and fix. Before we merge it, have you used HAProxy 1.8 without issues?

**Comment by valiant1x** [comment]
Fixed in 54bc700b2f9d8dad6e1aacd834c8431c0ded60bd

**Comment by zib92130** [comment]
@valiant1x Currently I didn't succeed to make the exit node working (I cannot connect myself on it and other user reported to fail too). I don't know if its related to HAProxy 1.8, but as this was a stopper for me for the installation, I shared the info and apply the fix to go on. Note that I haven't use the easy-deploy-node.sh script...
